### PR TITLE
Draft clarifications

### PR DESCRIPTION
**Pull request by dme26**
Collecting together a number of tweaks and rewording that likely require checking and possible reworking.

Any suggested breaking up, reorganisation or other edits welcome!

### CODE REVIEW

**Comment by sstevens2** [comment]
So far this looks good to me. I can't add comments since it is marked as draft and in `02-meet-docker.md` I might remove the "just" addition but otherwise I think the changes so far could be merged when ready.

File: _episodes/02-meet-docker.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ then you need to check that you have started the Docker Desktop, Docker Engine,
 ## Getting help
 Often when working with a new command line tool, we need to get help. These tools often have some
 sort of subcommand or flag (usually `help`, `-h`, or `--help`) that displays a prompt describing how to use the
-tool. For Docker, it's no different. If we run `docker --help`, we see the following output (only `docker` also works):
+tool. For Docker, it's no different. If we run `docker --help`, we see the following output (just running `docker` also produces the help message):
```
Comment (sstevens2) [inline _episodes/02-meet-docker.md:74]:
```suggestion
tool. For Docker, it's no different. If we run `docker --help`, we see the following output (running `docker` also produces the help message):
```

**Review by sstevens2 — APPROVED** [review]
One small change suggested but otherwise I think it looks good!

File: _episodes/02-meet-docker.md (hunk 1 above)
Comment (dme26) [reply _episodes/02-meet-docker.md:74]: Nice—even clearer!

**Comment by dme26** [comment]
Thanks @sstevens2 !

**Pull request merged**